StripeConnect/StripeConnect/Source/Internal/Webview/ConnectComponentWebViewController.swift

```diff
@@ -335,6 +342,9 @@ private extension ConnectComponentWebViewController {
                          appearance: .init(appearance: componentManager.appearance, traitCollection: self.traitCollection),
                          fonts: componentManager.fonts.map({ .init(customFontSource: $0) }))
         }))
+        addMessageHandler(FetchAppInfoMessageHandler.init(didReceiveMessage: { [weak self] _ in
+            return .init(applicationId: self?.bundleIdProvider() ?? "")
+        }))
```

Comment:
Should we instead only add this message handler if we have a bundle ID? 
```suggestion
        if let bundleId = bundleIdProvider() {
            addMessageHandler(FetchAppInfoMessageHandler.init(didReceiveMessage: { 
                return .init(applicationId: bundleId)
            }))
        }
```